README.md

```diff
@@ -1,4 +1,56 @@
-# ADS 508 Final Project
-Team Members: Shailja Somani, John Vincent Deniega, & Muris Saab\
-Spring 2024\
-University of San Diego
+# ADS-508 Team Project
```

Comment:
I think we should still keep team member names up here! (I saw you deleted the previous header, but at least names are still useful I think)

Reply:
done